### PR TITLE
Update dependency

### PR DESCRIPTION
**Pull request by RustySol**

### CODE REVIEW

File: lib/package.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
     "typescript": "^4.1.3"
   },
   "dependencies": {
-    "@liqnft/candy-shop-sdk": "0.4.7",
+    "@liqnft/candy-shop-sdk": "0.4.9",
```
Comment (haVincy) [inline lib/package.json:53]: Does this mean we will need to maintain the sdk with fixed version in `lib` as well?

File: lib/src/core/Orders/index.tsx [hunk 2]
```diff
@@ -179,20 +178,18 @@ export const Orders: React.FC<OrdersProps> = ({
                       </FlexItem>
                     ))}
                 </Flex>
-              ) : !loading && !orders.length ? (
-                <Empty description="No orders found" />
-              ) : (
+              ) : !orders.length ? (
```
Comment (haVincy) [inline lib/src/core/Orders/index.tsx:181]: This should be `orders.length ?`.

File: lib/package.json (hunk 1 above)
Comment (RustySol) [reply lib/package.json:53]: ideally yes, unless sdk has some major upgrade

File: lib/src/core/Orders/index.tsx (hunk 2 above)
Comment (RustySol) [reply lib/src/core/Orders/index.tsx:181]: let me rebase the commit, I think the history looks dirty

**Comment by RustySol** [comment]
Closing for #91

**Pull request closed without merging**